packages/marshal/README.md

```diff
@@ -5,35 +5,35 @@ objects) into a string, and back again.
 
 The `marshal` module helps with conversion of "capability-bearing data", in
 which some portion of the structured input represents "pass-by-proxy" or
-"pass-by-presence" objects. These should be serialized into markers that
+"pass-by-presence" objects. These should be toCapDatad into markers that
```

Comment:
Also, please search the project for all `toCapDatad` occurrences and replace with `serialized`.

```suggestion
"pass-by-presence" objects. These should be serialized into markers that
```